### PR TITLE
Improve look & feel of site

### PR DESCRIPTION
**Pull request by jasich**
- [ ] Add back in Google Analytics Tracker
- [ ] Remove unused pages (overview & generator)
- [ ] Move & cleanup inline JavaScript 
- [ ] Cleanup/Refactor CSS
- [ ] Make footer links work
- [ ] Extract common site theme to apply to other pages (e.g. latest)
- [ ] Get proper top & bottom text shown as templates change in generator
- [ ] Use more columns on wide screens
- [ ] Change typeahead text to a color better than white :person_frowning: 
- [ ] Copy the existing homepage content to the README or other docs (@jacebrowning)

### CODE REVIEW

**Comment by joshfriend** [comment]
:sparkles: :cake: :sparkles: 

If you don't specifically enter `_` as the text for top/bottom to make one blank, the generate button does nothing.

File: memegen/routes/future.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@blueprint.route("")
+@set_renderers(HTMLRenderer)
+def get():
+    imgs = [img for img in samples()]
+
+    [print(img) for img in imgs]
+
+    return render_template(
+        'future.html',
+        imgs=imgs
```
Comment (jacebrowning) [inline memegen/routes/future.py]: This can just be `imgs=samples()` when you're done with the debugging code above.

**Comment by jacebrowning** [comment]
@jasich I have given you push access to this repository. Would you mind pushing this code to an upstream branch so that I can help work on some of these checklist items?

**Comment by jasich** [comment]
Closing this in favor of: https://github.com/jacebrowning/memegen/pull/178

**Pull request closed without merging**